tmd/Tree/methods.py

```diff
@@ -184,15 +195,30 @@ def get_point_path_distances(self):
     return path_lengths
 
 
+def get_point_trunk_length(self):
```

Comment:
Yeah, I would not make this a "point" function but rather a "trunk_length" single value function.

Reply:
Ok